functions/routes/ecom/webhook.js

```diff
@@ -232,6 +231,117 @@ exports.post = async ({ appSdk, admin }, req, res) => {
                   message
                 })
               })
+          } else if (trigger.resource === 'products' && trigger.action === 'change') {
+            console.log('> Edit product ', resourceId, 's: ', storeId)
+
+            let query = 'status!=cancelled&transactions.type=recurrence'
+            query += '&transactions.app.intermediator.code=galaxpay_app'
+
+            query += `&items.product_id=${resourceId}`
+
+            try {
+              const { result } = await getOrderWithQueryString(appSdk, storeId, query, auth)
+
+              if (result && result.length) {
+                const galaxPaySubscriptions = await getSubscriptionsByListMyIds(
+                  galaxpayAxios,
+                  result.reduce((orderIds, order) => {
+                    orderIds.push(order._id)
+                    return orderIds
+                  }, [])
+                )
+
+                if (galaxPaySubscriptions && galaxPaySubscriptions.length) {
+                  for (let i = 0; i < galaxPaySubscriptions.length; i++) {
+                    const subscription = galaxPaySubscriptions[i]
+                    try {
+                      const order = await findOrderById(appSdk, storeId, subscription.myId, auth)
+                      const product = await getProductsById(appSdk, storeId, resourceId, auth)
+
+                      if (order) {
+                        const docSubscription = await getDocSubscription(order._id, collectionSubscription)
+
+                        order.items.forEach(orderItem => {
+                          if (orderItem.product_id === product._id) {
+                            if (orderItem.variation_id) {
+                              const variation = product.variations.find(itemFind => itemFind._id === orderItem.variation_id)
```

Comment:
```suggestion
                              const variation = product.variations.find(itemFind => itemFind.sku === orderItem.sku)
```

O ID da variação pode mudar, em geral o SKU é mantido